README.md

```diff
@@ -1 +1,50 @@
-PlanGrid .NET SDK for the public PlanGrid API
\ No newline at end of file
+#PlanGrid .NET SDK for the public PlanGrid API
+
+To install, use the Nuget [package](https://www.nuget.org/packages/PlanGrid.Api/):
+
+    Install-Package PlanGrid.Api
+	
+##Getting Started
+
+Once installed, to access the api, simply use `PlanGridClient.Create()`.  
+However, you will need to provide your API key to gain access.  There are 
+three ways to provide the key.  The simplest is to pass it directly as an
+argument to `Create`:
+
+    IPlanGridApi api = PlanGridClient.Create("yourapikey");
+    
+This has the advantage of being straightforward, but has the disadvantage of
+baking the key into your code, which reduces security and makes it more 
+difficult to update.
+	
+Alternatively, you can set the key in you `App.config` (or `Web.config`) file: 
+
+    <appSettings>
+        <add key="PlanGridApiKey" value="yourapikey" />
+    </appSettings>
+
+Ensure that you do not already have an `<appSettings>` node -- if you do, add 
+the `<add />` element to the existing one.  When using this approach, you can
+create the client with the parameterless constructor:
+
+    IPlanGridApi api = PlanGridApiClient.Create();
+    
+Finally, you may set the key in an environment variable; the variable name 
+must be `PLANGRIDAPIKEY`.
+
+##Getting a list of projects
+
+Having provided the API key in one of the ways outlined above, you can now
+interact with the variou endpoints available to you.  To start, we'll simply
+get a list of projects:
+
+    Page<Project> projects = await api.GetProjects();
+    
+Some things to note:
+* This returns a paginated subset of all the projects to which you have access.
+* The default limit is 50 projects
+* By defualt, zero projects are skipped.
```

Comment:
`By defualt, zero projects are skipped` 
IMO:  while completely straightforward, stating this is kind of weird. maybe leave this out or reword it.